src/pokemanki/gui/pokemanki_trade.py

```diff
@@ -0,0 +1,197 @@
+        self.dialog.resize(400, 472)
+        self.dialog.setMinimumSize(QtCore.QSize(250, 100))
+        self.verticalLayout = QtWidgets.QVBoxLayout(self.dialog)
+        self.verticalLayout.setSizeConstraint(
+            QtWidgets.QLayout.SizeConstraint.SetMaximumSize
+        )
+        self.verticalLayout.setSpacing(9)
+        self.verticalLayout.setObjectName("verticalLayout")
+        self.title = QtWidgets.QLabel(self.dialog)
+        self.title.setMaximumSize(QtCore.QSize(16777215, 123))
+        font = QtGui.QFont()
+        font.setPointSize(20)
+        font.setBold(True)
+        self.title.setFont(font)
+        self.title.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
+        self.title.setWordWrap(True)
+        self.title.setObjectName("title")
+        self.verticalLayout.addWidget(self.title, 0, QtCore.Qt.AlignmentFlag.AlignTop)
+
+        self.webView = AnkiWebView(parent=self.dialog, title="pokémanki trades")
+        sizePolicy = QtWidgets.QSizePolicy(
+            QtWidgets.QSizePolicy.Policy.Expanding,
+            QtWidgets.QSizePolicy.Policy.Expanding,
+        )
+        sizePolicy.setVerticalStretch(0)
+        sizePolicy.setHeightForWidth(self.webView.sizePolicy().hasHeightForWidth())
+        self.webView.setSizePolicy(sizePolicy)
+        self.webView.setObjectName("webView")
+        self.webView.set_bridge_command(parent.on_bridge_cmd, parent)
+        self.verticalLayout.addWidget(self.webView)
+
+        self.buttonBox = QtWidgets.QDialogButtonBox(self.dialog)
+        self.buttonBox.setOrientation(QtCore.Qt.Orientation.Horizontal)
+        self.buttonBox.setStandardButtons(
+            QtWidgets.QDialogButtonBox.StandardButton.Close
+        )
+        self.buttonBox.setCenterButtons(False)
+        self.buttonBox.setObjectName("buttonBox")
+        self.verticalLayout.addWidget(self.buttonBox)
+
+        self.retranslateUi()
+        self.buttonBox.accepted.connect(self.dialog.accept)  # type: ignore
+        self.buttonBox.rejected.connect(self.dialog.reject)  # type: ignore
+        QtCore.QMetaObject.connectSlotsByName(self.dialog)
+
+        # Set in the middle of the screen
+        width = 800
+        height = self.dialog.height() + 150
+
+        if platform.system() == "Windows":
+            user32 = ctypes.windll.user32
+            posX = int(user32.GetSystemMetrics(0) / 2 - width / 2)
+            posY = int(user32.GetSystemMetrics(1) / 2 - height / 2)
+        else:
+            posX = int(mw.frameGeometry().width() / 2 - width / 2)
+            posY = int(mw.frameGeometry().height() / 2 - height / 2)
+
+        self.dialog.setGeometry(posX, posY, width, height)
+
+    def retranslateUi(self):
+        _translate = QtCore.QCoreApplication.translate
+        self.dialog.setWindowTitle(_translate("Dialog", "Trade Pokémon"))
+        self.title.setText(_translate("Dialog", "Today's Trades"))
+
+    def setupWebView(self, trades):
+        """
+        Set up the web view's html.
```

Comment:
Put the summary of the docstring directly after the triple quotes.
[PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings)
```suggestion
        """Set up the web view's html.
```